src/content/ui-api/kubectl-gs/update-cluster.md

```diff
@@ -0,0 +1,47 @@
+---
+linkTitle: update cluster 
+title: "'kubectl gs update cluster' command reference"
+description: Reference documentation on how to update a Cluster using 'kubectl gs'.
+weight: 100
+menu:
+  main:
+    parent: uiapi-kubectlgs
```

Comment:
```suggestion
    identifier: kubectl-gs-update-cluster
    parent: uiapi-kubectlgs
```

This is required in this case to make the item appear in the menu. Without this, HUGO issues a warning

```
WARN 2021/12/14 11:11:09 "/Users/marian/git/giantswarm/docs/src/content/ui-api/kubectl-gs/update-cluster.md:1:1": duplicate menu entry with identifier "update cluster" in menu "main"
```

and does not add the page to the menu.